tst/EnergyPlus/unit/OutputReportTabularAnnual.unit.cc

```diff
@@ -272,3 +272,40 @@ TEST_F(EnergyPlusFixture, OutputReportTabularAnnual_columnHeadersToTitleCase)
 
 }
 
+TEST_F( EnergyPlusFixture, OutputReportTabularAnnual_invalidAggregationOrder )
+{
+	std::string const idf_objects = delimited_string( {
+		"Version,8.7;",
+		"Output:Table:Annual,",
+		"Test Report, !- Name",
+		", !- Filter",
+		", !- Schedule Name",
+		"Electricity:Facility, !- Variable or Meter 2 Name",
+		"SumOrAverageDuringHoursShown, !- Aggregation Type for Variable or Meter 2",
+		", !- field Digits After Decimal 2",
+		"Misc Facility Electric Energy, !- Variable or Meter 3 Name",
+		"SumOrAverage, !- Aggregation Type for Variable or Meter 3",
+		"0; !- field Digits After Decimal 3",
+	} );
```

Comment:
Do you think it's worth adding a test for `Output:Table:Monthly`?  It's hard to tell from the diff exactly what functional changes were made to the code, so the more unit tests the better.